src/entities/activity/ui/items/CheckboxItem/PortraitCheckboxOption.tsx

```diff
@@ -0,0 +1,96 @@
+import { useMemo } from 'react';
+
+import { Box } from '~/shared/ui';
+import {
+  CheckboxItem,
+  CustomTooltip,
+  SelectBaseBox,
+  SelectBaseImage,
+  SelectBaseText,
+} from '~/shared/ui';
+import { cutString } from '~/shared/utils';
+
+type Props = {
+  id: string;
+  name: string;
+  value: string | number;
+  label: string;
+
+  description: string | null;
+  image: string | null;
+  disabled?: boolean;
+  defaultChecked?: boolean;
+  color: string | null;
+
+  onChange: () => void;
+  replaceText: (value: string) => string;
+};
+
+export const PortraitCheckboxOption = (props: Props) => {
+  const tooltipText = useMemo(() => {
+    if (props.description) {
+      return props.replaceText(props.description);
+    }
+
+    return null;
+  }, [props]);
+
+  const labelText = useMemo(() => {
+    return props.replaceText(props.label);
+  }, [props]);
+
+  const cutTheTextBasedOnOptionParams = (text: string) => {
+    if (props.image && !tooltipText) {
+      return cutString(text, 21);
+    }
+
+    if (props.image && tooltipText) {
+      return cutString(text, 12);
+    }
+
+    return cutString(text, 25);
+  };
+
+  const hasImage = !!props.image;
+
+  return (
+    <SelectBaseBox
+      color={props.color}
+      onHandleChange={props.onChange}
+      checked={props.defaultChecked}
+      padding="8px 8px"
+      sx={{ minHeight: '188px', width: '148px', alignItems: !hasImage ? 'center' : undefined }}
```

Comment:
Width should be defined and controlled by the Grid component

Reply:
Done